src/utils/duration-humanizer.ts

```diff
@@ -0,0 +1,39 @@
+import * as humanizeDuration from 'humanize-duration';
+
+const getTimeInText = (position: number, uiConfig: any) => {
+  try {
+    const durationHumanizer = getDurationHumanizer(uiConfig);
+    if (durationHumanizer) {
+      try {
+        return position ? durationHumanizer(position * 1000) : '';
```

Comment:
maybe rename position to something like "seconds" or "currentTime" or  so it'll be more explicit that it's in seconds. in case we want to make it more generic later and handle both seconds and milliseconds.